go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	k8s.io/code-generator v0.24.0
 	k8s.io/gengo v0.0.0-20211129171323-c02415ce4185
 	k8s.io/klog/v2 v2.60.1
+	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
```

Comment:
This means everyone who depends on the API repo will now depend on this, it would be better to create a separate go module `github.com/openshift/api/tools` to store our tooling to prevent other projects from having these excess dependencies.

I've started some work on that in a local branch to work towards this

Reply:
> This means everyone who depends on the API repo will now depend on this

apimachinery already depends on kube-openapi: https://github.com/kubernetes/apimachinery/blob/master/go.mod#L25, so this isn't new.